### DISCUSSION TITLE
Inconsistent behavior when running scans against a large number of targets

### DISCUSSION THREAD

**Discussion by cfsdes** [projectdiscovery/nuclei#868] [General]
Hello,

I'm trying to run nuclei with a big input (200.000 targets) and it's running but without returning results.

When I run only on specific target, it works and return vulnerability, but with big input not.

Syntax used:
nuclei -l targets.txt -t templates/ -metrics

**Comment by geeknik** [comment]
I'm currently running a test against 12,753,476 endpoints with 4 templates and results are being returned. 

My command line is pretty simple:
`nuclei -r $RESOLVERS -l $HOSTS -t $TEMPLATES -o $OUT`

Please make sure that your version of nuclei is up to date version 2.4.0 is current as of this post. Also, you can use the `-debug` and/or `-v` flag to ensure that everything is running okay.

**Comment by cfsdes** [comment]
What resolvers wordlist are you using? Why it's necessary?

**Reply by geeknik** [reply]
It's just a bunch of public resolvers, nothing special in there, stuff like Google, Quad9, Verizon, etc.

**Reply by cfsdes** [reply]
Thank you very much, I'll try it
